docs/admin/authentication.md

```diff
@@ -81,6 +81,59 @@ provider external to Kubernetes.  We plan to make it easy to develop modules
 that interface between Kubernetes and a bedrock authentication provider (e.g.
 github.com, google.com, enterprise directory, kerberos, etc.)
 
+## APPENDIX
+
+### Make Certificates
+
+When using client certificate authentication, you should generate certificates by
+deployment script or some commands.
+
+**Deployment script** is implemented at
+`cluster/saltbase/salt/generate-cert/make-ca-cert.sh`.
+Execute this script with two parameters. First is the `cert_ip`, second is the
+`subject-alt-name`. `cert_ip` is used to generate the CA default CN and append to
+`subject-alt-name`. ([subjectAltName](http://wiki.cacert.org/FAQ/subjectAltName))
+This script will make a ca.crt, a server.cert and a server.key.
+Finally, add these parameters
+`--client-ca-file=/srv/kubernetes/ca.crt`
+`--tls-cert-file=/srv/kubernetes/server.cert`
+`--tls-private-key-file=/srv/kubernetes/server.key`
+into apiserver start parameters.
+
+**Easyrsa** is an easy way to make certificates by hands.
+1. Download the easyrsa and do some preparations.
```

Comment:
(here and below) You can use "1." for each numbered item. This makes it much easier to reorder the steps later if they change.